platform/lib/routers/pages.js

```diff
@@ -144,16 +148,40 @@ if (config.environment === 'development') {
 }
 
 if (config.environment !== 'development') {
```

Comment:
Doesn't this mean that filtering only work on non dev mode?

Reply:
That's right. Filtering for development is handled a bit differently as it involves rewriting the proxied response from Grow. See line 102 and onwards.